deepchecks/nlp/checks/model_evaluation/train_test_performance.py

```diff
@@ -33,6 +33,17 @@ class TrainTestPerformance(TrainTestPerformanceAbstract, TrainTestCheck):
     scorers: Union[Mapping[str, Union[str, Callable]], List[str]], default: None
         Scorers to override the default scorers, find more about the supported formats at
         https://docs.deepchecks.com/stable/user-guide/general/metrics_guide.html
+    min_samples: int, default: 30
+        Minimum number of samples required to calculate model performance. If scorer is per class, then
+        this is the minimum number of samples per class.
+    n_top_classes: int, default: 10
+        Number of top classes to show in the label graphs. The top classes are determined by the show_classes_by
+        parameter. if None, then all classes are shown.
+    show_classes_by: str, default: 'test_largest'
```

Comment:
Maybe also option of largest diff?

Reply:
Also, the issue is - by which metric? And for each graph separately? It really makes it complicatd :/